flake.lock

```diff
@@ -24,10 +24,10 @@
       },
       "locked": {
         "lastModified": 1628586117,
-        "narHash": "sha256-8hS4xy7fq3z9XZIMYm4sQi9SzhcYqEJfdbwgDePoWuc=",
+        "narHash": "sha256-HSNezMKSYEwk5Ac3Tb7DaSUdYrqttk1WILrlzSqwaBw=",
         "owner": "NixOS",
         "repo": "nix",
-        "rev": "a6ba313a0aac3b6e2fef434cb42d190a0849238e",
+        "rev": "5ee937523da63b26cc6dd264cc3cb2717321c5cd",
```

Comment:
I made the base branch of https://github.com/NixOS/nix/pull/5949 such that there is no other new stuff in here

Reply:
Yay, https://github.com/NixOS/nix/pull/5949 is now merged.

Reply:
I'm anxious about merging this as-is, ideally we'll be able to update to a newer Nix master.